cli/ballerina-cli/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -53,14 +53,18 @@ OPTIONS
            and (`--cloud=docker`) for Docker.
 
        --list-conflicted-classes
-       		List the conflicting classes of conflicting JARs in the package.
+           List the conflicting classes of conflicting JARs in the project.
+
+       --sticky
```

Comment:
sticky flag is added with https://github.com/ballerina-platform/ballerina-lang/pull/34674. Hence we can remove this.